client/my-sites/google-my-business/select-location/location.js

```diff
@@ -0,0 +1,98 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import Gridicon from 'gridicons';
+import { localize } from 'i18n-calypso';
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import CompactCard from 'components/card/compact';
+import { getSelectedSiteSlug } from 'state/ui/selectors';
+import { recordTracksEvent } from 'state/analytics/actions';
+
+class GoogleMyBusinessLocation extends Component {
+	static propTypes = {
+		location: PropTypes.shape( {
```

Comment:
We could probably define a type, instead of redefining it in each component.
Since we don't have the source data yet I'm fine with keeping it as it is for now